src/app/work-item/work-item.service.ts

```diff
@@ -54,26 +66,35 @@ export class WorkItemService {
       .get(url, { headers: this.headers })
       .toPromise()
       .then(response => {
+        // Build the user - id map
         this.buildUserIdMap();
+        let wItems: WorkItem[];
         if (process.env.ENV == 'inmemory') {
           // Exclusively for in memory testing
-          this.workItems = response.json().data as WorkItem[];
-          this.workItems = this.workItems.reverse();
```

Comment:
Reversing has been there from the begging. Once we have ordered data from backend we can make it cleaner.